### PR TITLE
KK version upgrade

### PR DESCRIPTION
**Pull request by jaspnas**
Version upgrade for KK

Includes changes from #286, #290, #299, #302, #306 and #310.

### CODE REVIEW

File: date/functions.py [hunk 1]
```diff
@@ -11,7 +11,3 @@ def slugify_max(text, max_length=50):
         return trimmed_slug
     # First word is > max_length chars, so we have to break it
     return slug[:max_length]
-
-
-def days_hence(days):
-    return timezone.now() + timezone.timedelta(days=days)
```
Comment (JoarSabel) [inline date/functions.py:17]: image
What the hell was this

File: archive/mass_upload.py
Comment (JoarSabel) [inline archive/mass_upload.py:1]: Spoke with @jaspnas and it seems there are no references to mass_upload.py in the code, in which case, if this is a script. should we move it to scripts at some point?

**Review by JoarSabel — APPROVED** [review]
Looks good to me, should test on QA 

image

File: date/functions.py (hunk 1 above)
Comment (jaspnas) [reply date/functions.py:17]: Removed because pycharm showed it not having usages

File: archive/mass_upload.py
Comment (jaspnas) [reply archive/mass_upload.py:1]: Moved to #319

**Pull request merged**